app/models/user.rb

```diff
@@ -947,10 +951,6 @@ def editable_fields_of_user(user)
     fields += editable_personal_preference_fields(user)
     fields += editable_competitor_info_fields(user)
     fields += editable_avatar_fields(user)
-    # Delegate Status Fields
-    if admin? || board_member? || senior_delegate?
-      fields += %i(delegate_status region_id location)
-    end
```

Comment:
If you are removing them here, have you made sure that the actual corresponding form (that would have checked for the presence of these three values) is also gone?

Reply:
Actually the three values are not used in the edit user form. They have been already moved to separate form and is handled differently.